src/js/facility-locator/containers/VAMap.jsx

```diff
@@ -89,7 +96,14 @@ class VAMap extends Component {
     const { currentQuery } = prevProps;
     const newQuery = this.props.currentQuery;
 
-    if (isEmpty(this.props.facilities) && !newQuery.inProgress && currentQuery.inProgress && newQuery.bounds && parseInt(newQuery.zoomLevel, 10) > 2) {
+    if (
+      isEmpty(this.props.facilities) &&
+      !newQuery.inProgress &&
+      currentQuery.inProgress &&
+      newQuery.bounds &&
+      parseInt(newQuery.zoomLevel, 10) > 2 &&
+      !newQuery.error
```

Comment:
Usually I like to assign long conditions to descriptively named variables like `shouldUpdateSearchQuery`.